provisioning/containers/blender/autolimits.py

```diff
@@ -0,0 +1,21 @@
+import os
+import bpy
+from bpy.app.handlers import persistent
+
+
+@persistent  # Keep across file reloads
+def cpulimits_setter(_):  # An argument has to be present for handlers
+    try:
+        maxThreads = int(float(os.environ['CROWNLABS_CPU_LIMTIS']))  # Round possible floats
```

Comment:
Would round up, to avoid issues in case of limits <1CPU (and avoid wasting available resources if limits are not integers).